Marlin/src/lcd/dogm/dogm_Statusscreen.h

```diff
@@ -1370,7 +1370,7 @@
     #define STATUS_LOGO_X 0
   #endif
   #ifndef STATUS_LOGO_Y
-    #define STATUS_LOGO_Y _MIN(0U, ((20 - (STATUS_LOGO_HEIGHT)) / 2)
+    #define STATUS_LOGO_Y _MIN(0U, (20 - STATUS_LOGO_HEIGHT) / 2)
```

Comment:
I think it's better to enclose STATUS_LOGO_HEIGHT into brackets`(20 - (STATUS_LOGO_HEIGHT)) / 2)` you never know how it's defined

Reply:
Although it works with current code, your probably right, who know what someone will set it to in the future...